README.md

```diff
@@ -29,14 +36,18 @@ All test cases and parsers coverage can be found [here](https://asyncapi.github.
 ## Installation
 
 ### NodeJS
+
 ```bash
 npm install @asyncapi/specs
+yarn add @asyncapi/specs
```

Comment:
I'm not super-familiar with `yarn` - are we saying that `yarn` is required to use this module? Or that it's an option?

Reply:
Yarn is another Node package manager, created by Facebook, next to the npm (default one installed by NodeJS), but a lot of people use it because it has a lot of features that npm hasn't (but latest versions have similar functionality). It has a another CLI API to install packages so yarn is a option, how to install package by this manager.

Should we describe it as:

```bash
npm install @asyncapi/specs
// OR by Yarn
yarn add @asyncapi/specs
```

wdyt?

Reply:
Done :)